src/Concordium/Client/Types/Contract/Schema.hs

```diff
@@ -1,6 +1,18 @@
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE QuasiQuotes #-}
-module Concordium.Client.Types.Contract.Schema where
+module Concordium.Client.Types.Contract.Schema(ContractSchema(..),
+                                               Fields(..),
+                                               FuncName(..),
+                                               ModuleSchema(..),
+                                               SchemaType(..),
+                                               SizeLength(..),
+                                               decodeEmbeddedSchema,
+                                               decodeEmbeddedSchemaAndExports,
+                                               decodeModuleSchema,
+                                               getListOfWithKnownLen,
+                                               getListOfWithSizeLen,
+                                               lookupSignatureForFunc,
+                                               putLenWithSizeLen) where
```

Comment:
Is this really the best possible indentation?

I would recommend not having all this whitespace on the left.